### ISSUE TITLE
physics demo loss function

### ISSUE THREAD

**Issue by kingwmk** [deepmind/graph_nets#25]
The calculation of the loss function, why is the difference between the nodes' output of the model and the velocity dimension's feature, that is, why the output of the node is linearly transformed into two-dimensional.

**Comment by alvarosg** [comment]
The model is trained to predict the two dimensional velocity of each node/particle, doing supervised learning on the ground truth velocity, which is why the loss minimizes the squared difference between the output and the velocity. Given the velocity, the predicted position is then calculated as `new_position = previous_position + predicted_velocity*timestep`.

**Comment by alvarosg** [comment]
Closing due to inactivity, please feel free to reopen if necessary.
